components/expenses/ProcessExpenseButtons.js

```diff
@@ -57,7 +58,8 @@ export const hasProcessButtons = permissions => {
     permissions.canPay ||
     permissions.canMarkAsUnpaid ||
     permissions.canMarkAsSpam ||
-    permissions.canDelete
+    permissions.canDelete ||
+    permissions.canMarkAsIncomplete
```

Comment:
Shouldn't that be:

```suggestion
    (permissions.canMarkAsIncomplete && displayMarkAsIncomplete)
```

Reply:
It should be, but since this is not relevant to the possible conditions and we're moving this to the actions button I'm fine with not refactoring this function and its calls.